dev/forbidden_from_release_tests/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: forbidden_from_release_tests
+publish_to: 'none'
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+dependencies:
+  args: 2.0.0
+  file: 6.1.0
```

Comment:
If you're not writing unit tests for your tests, I don't think using file or process buys you anything. Its almost exactly the same API as Process.run/dart:io File.

Reply:
I figure it makes it easier in the future and no one will have to rerun update-packages. I think these packages also have some additional checks/guards that I'd miss using dart:io directly too right?

Reply:
not really :)